doc/development/developers_guide.md

```diff
@@ -0,0 +1,66 @@
+# Developers Guide
+
+This page contains information for developers.
+
+## Integration tests
+For the integration tests you can select between three different test environments:
+
+|Setup                          |Description                                |
+|-------------------------------|-------------------------------------------|
+| `testcontainers` (default)    | For each integration test class a separate docker container is started. This method provides best isolation. It is used by the CI tests.|
+|`local`                        | This method is faster that `testcontainers` since it reuses the same container all the time. You must start the container by hand by running `docker-compose up` in `src/test/resources/`| 
```

Comment:
```suggestion
|`local`                        | This method is faster than `testcontainers` since it reuses the same container all the time. You must start the container manually by running `docker-compose up` in `src/test/resources/`| 
```